app/models/user.rb

```diff
@@ -211,6 +211,7 @@ def sync_profile
 
   scope :eager_load_serialized_data, -> { includes(:roles) }
   scope :registered, -> { where(registered: true) }
+  scope :with_feed, -> { where.not(feed_url: [nil, ""]) }
```

Comment:
❤️

Reply:
Given that we have quite a few users, do you think it could make sense to add a [partial index](https://use-the-index-luke.com/sql/where-clause/partial-and-filtered-indexes) for this? 

```sql
CREATE INDEX feed_url_present_id
ON users (feed_url)
WHERE coalesce(feed_url, '') <> '';
```